code/game/objects/structures/extinguisher.dm

```diff
@@ -48,6 +51,27 @@
 		opened = !opened
 	update_icon()
 
+/obj/structure/extinguisher_cabinet/attackby(obj/item/W as obj, mob/user as mob, params)
```

Comment:
Holy duplicate proc declaration Batman!

See line 17 and move this stuff into that.